components/costs/README.md

```diff
@@ -0,0 +1,7 @@
+# Cost and usage reports
+[Requirements](https://github.com/superwerker/superwerker/issues/30)
```

Comment:
I don't think we need a dedicated `README` here but we could add an ADR document (see e.g. #18, #33 , #35)

Reply:
Done